docs/otel/sdk-metrics.md

```diff
@@ -882,5 +887,135 @@ E.g. for Java the fully qualified classname SHOULD be used in this case.
 <!-- END AUTOGENERATED TEXT -->
 <!-- endsemconv -->
 
+## Operation Metrics
+
+### Metric: `otel.sdk.exporter.operation.duration`
```

Comment:
is the idea for this to include metric exporter durations as well?

Reply:
I believe so.